benchexec/tools/swat.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+from benchexec.tools.template import BaseTool2
+
+
+class Tool(BaseTool2):
+    """
+    Tool info module for SWAT a tool ensemble for the dynamic symbolic execution of modern Java.
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("run-swat.sh")
+
+    def version(self, executable):
+        return '42'
+
+    def name(self):
+        return "SWAT"
+
+    def cmdline(self, executable, options, task, rlimits):
+        cmd = [executable] + options
+        if task.property_file:
+            cmd.append(task.property_file)
+        return cmd + list(task.input_files)
```

Comment:
Does your tool support tasks that consist of more than one input file? If yes, this is fine, otherwise please use `task.single_input_file` such that users immediately get a proper error.

Reply:
As we are in the Java context, we retrieve multiple arguments, i.e. the path to the common folder and the SUT itself. 
Upon refractoring I get the following error: `Tool does not support tasks with more than one input file`. So I assume this is correct.